src/classes/UTIL_AbstractCommon_BATCH_TEST.cls

```diff
@@ -0,0 +1,472 @@
+/*
+    Copyright (c) 2019 Salesforce.org
+    All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Utilities
+* @description Tests for UTIL_AbstractCommon_BATCH class
+*/
+@isTest(isParallel=true)
+public class UTIL_AbstractCommon_BATCH_TEST extends UTIL_AbstractCommon_BATCH {
+
+    /*******************************************************************************************************************
+     * @description The number of Accounts to create for test conditions
+     */
+    private static final Integer TEST_ACCOUNT_CNT = 7;
+
+    private Integer expectedResult;
+    private Integer testChunkSize = 1;
+    private String testWhereClause = '';
+
+    /*******************************************************************************************************************
+     * @description Test Batch Execution with inner test batch class to test non chunking conditions
+     */
+    @isTest
+    public static void executeTestBatch() {
+
+        List<Account> accs = new List<Account>();
+
+        for (Integer i = 0; i < TEST_ACCOUNT_CNT; i++) {
+            accs.add(
+                new Account(
+                    Name = '1-Test Account #' + i
+                )
+            );
+        }
+        insert accs;
+
+        Test.startTest();
+
+        UTIL_AbstractCommon_BATCH_TEST tab = new UTIL_AbstractCommon_BATCH_TEST(10);
```

Comment:
@force2b - I wanted to...it is not supported/allowed.  This monstrosity was the best i could do, short of a separate class.  It gave a wacky error, that it turned out to be was because it was an inner class.

Reply:
Might be because the Abstract class implements Batchable and Schedulable, and those cannot be implemented in an Inner class. What about moving this into a different class - `UTIL_AbstractCommon_MOCK extends UTIL_AbstractCommon_BATCH` and then keep just the test methods in this class?